x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form_objectives_timeslices.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFieldNumber, EuiFlexGrid, EuiFlexItem, EuiFormLabel } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Control, Controller } from 'react-hook-form';
+import type { CreateSLOParams } from '@kbn/slo-schema';
+
+export interface SloEditFormObjectivesTimeslicesProps {
```

Comment:
nit: should we call it simply `Props` since we only have one per file/component. When importing it elsewhere, we can always name it `Props as MyComponentProps`.

Reply:
Ok